tools/wt_binary_decode.py

```diff
@@ -440,22 +522,34 @@ def print_timestamps(p, b, extra):
     WT_CELL_TS_STOP = 0x10
     WT_CELL_TXN_START = 0x20
     WT_CELL_TXN_STOP = 0x40
-        
+    
     if extra != 0:
         p.rint('cell has timestamps:')
         if extra & WT_CELL_PREPARE != 0:
             p.rint(' prepared')
         if extra & WT_CELL_TS_DURABLE_START != 0:
+            pagestats.start_ts_sz += discard_unpacked_uint64(b)
```

Comment:
Probably should exclude the the durable start and stop, not needed in the immediate case.

Reply:
Tell me more, I don't think I understand the why? Aren't these also fields that don't exist in 4.2 and potentially get removed when we rewrite a page?

Reply:
They don't exist in 4.2 and are for prepared updates which could be a separate set of stats (I don't think they should be bundled in).

Reply:
Broke these out into their own counters, but didn't report them yet. Should I add columns for them and include them in the totals or just leave it as is?

Reply:
Add them in to be complete.